client/Components/User/User.jsx

```diff
@@ -1,39 +1,78 @@
 import React, { Component } from "react";
 import styles from "./User.css";
+import Modal from "../Modal/Modal.jsx";
+import Links from "../Links/Links.jsx";
 class User extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      isModalOpen: false
+    };
+    this.closeModal = this.closeModal.bind(this);
+    this.openModal = this.openModal.bind(this);
+  }
+
+  // close modal (set isModalOpen, true)
+  closeModal() {
+    this.setState({
+      isModalOpen: false
+    });
+  }
+
+  // open modal (set isModalOpen, false)
+  openModal() {
+    this.setState({
+      isModalOpen: true
+    });
   }
 
   render() {
     return (
-      <div className={styles.users}>
-        <div className={styles.image}>
-          <img src={this.props.user.profile_image} className={styles.avatars} />
+      <div id={styles.usersContainer}>
```

Comment:
Same as previous comment, but this one seems more complex to map. Just throwing out the idea. Lmk if it's not possible.